addons/mail/static/src/core/web/chatter.xml

```diff
@@ -21,8 +21,7 @@
                     Log note
                 </button>
                 <div class="flex-grow-1 d-flex">
-                    <button t-if="props.hasActivities" class="o-mail-Chatter-activity btn btn-secondary text-nowrap" t-att-class="{ 'my-2': !props.compactHeight }" data-hotkey="shift+a" t-on-click="scheduleActivity">
-                        <i class="fa fa-clock-o me-1"/>
+                    <button t-if="props.hasActivities" class="o-mail-Chatter-activity btn btn-secondary ms-2 text-nowrap" t-att-class="{ 'my-2': !props.compactHeight }" data-hotkey="shift+a" t-on-click="scheduleActivity">
```

Comment:
I would use `ms-3` instead of `ms-2` to better distinguish the Activites button. By using `ms-2`, it looks like there's a spacing issue.

| ms-2 |
|--------|
| ![Capture d’écran 2023-12-29 à 09 23 24](https://github.com/odoo/odoo/assets/80679690/a4eacf5a-91c9-4b01-907c-0c359cbb281a) |
| ms-3 |
| ![Capture d’écran 2023-12-29 à 09 22 29](https://github.com/odoo/odoo/assets/80679690/9d6c44ab-04f1-4759-837f-e9b013eb74ca) | 

What do you think ?